rts/lra/lra-coordinator/pom.xml

```diff
@@ -30,7 +30,9 @@
             <plugin>
                 <groupId>io.thorntail</groupId>
                 <artifactId>thorntail-maven-plugin</artifactId>
-                <version>${version.wildfly-thorntail}</version>
+<!--
+                <version>${version.thorntail}</version>
+-->
```

Comment:
Do we need to keep this commented out?

Reply:
I'm not ready yet - I will clean up the PR when I've seen the TCK passing